packages/scandipwa/src/util/Validator/Config.js

```diff
@@ -90,5 +92,6 @@ export const VALIDATION_RULES = {
     [VALIDATION_INPUT_TYPE.date]: /\d{4}-\d{1,2}-\d{1,2}/,
     [VALIDATION_INPUT_TYPE.phone]: /^[\\+]?[(]?[0-9]{3}[)]?[-\s\\.]?[0-9]{3}[-\s\\.]?[0-9]{4,6}$/im,
     // eslint-disable-next-line max-len
-    [VALIDATION_INPUT_TYPE.emailList]: /^(([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\.([a-zA-Z]{2,5}){1,25})+([,\s]+(([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\.([a-zA-Z]{2,5}){1,25})+)*$/
+    [VALIDATION_INPUT_TYPE.emailList]: /^(([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\.([a-zA-Z]{2,5}){1,25})+([,\s]+(([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\.([a-zA-Z]{2,5}){1,25})+)*$/,
+    [VALIDATION_INPUT_TYPE.fileExtension]: /\w+\.png$|\w+\.jpg$/i
```

Comment:
As we discussed in private, rename this validation to be more precise. Currently with that name we could apply this validation on other file types like documents.
Change it to something like `imageFile` so it's clear that this validation is only meant for images
Same goes for the message, etc. 
Adjust it across Your code changes